uaclient/sso.py

```diff
@@ -9,23 +9,12 @@
 
 
 API_PATH_V2 = '/api/v2'
-API_PATH_USER_KEYS = API_PATH_V2 + '/keys/'
-API_PATH_OAUTH_TOKEN = API_PATH_V2 + '/tokens/oauth'
 API_PATH_TOKEN_DISCHARGE = API_PATH_V2 + '/tokens/discharge'
-API_PATH_TOKEN_REFRESH = API_PATH_V2 + '/tokens/refresh'
 
 
 # Some Ubuntu SSO API responses use UNDERSCORE_DELIMITED codes, others use
 # lowercase-hyphenated. We'll standardize on lowercase-hyphenated
 API_ERROR_2FA_REQUIRED = 'twofactor-required'
-API_ERROR_2FA_FAILURE = 'twofactor-failure'
```

Comment:
I feel like this and API_ERROR_INVALID_DATA are premature too as I'd expect them to be handled by a branch that should be fixing issue: #504

Reply:
These aren't used in the codebase today, and I already get out relevant error messages:

```
$ UA_CONFIG_FILE=uaclient-devel.conf fakeroot ua attach
Email: daniel.watkins@canonical.com
Password: 
Second-factor auth: 123456
ERROR: HTTP Error 403: Forbidden: The provided 2-factor key is not recognised.
Could not attach machine. Unable to obtain authenticated user token
```

The issue in #504 isn't that we don't have useful content in the error, it's that the errors are needlessly verbose/noisy. So it isn't clear to me why we _would_ need them (and if I'm wrong, the branch that introduces the code that needs them can always reintroduce them!).

Reply:
The branch you are working for this may not actually have to use them.  It's probably a cleaner approach than the attempt I made with https://paste.ubuntu.com/p/HdZXjHHCrx/ which tried to map error types from SSO endpoints a bit better.  As you mention, it's simple enough to reintroduce if needed.